services/sources/source_service.py

```diff
@@ -10,9 +10,8 @@ class SourceService(ABC):
     @abstractmethod
     def get_records(
         self,
-        full_import: bool=False,
```

Comment:
Since we don't have this full_import parameter anymore what is the refactor to replace the use of it?

Reply:
basically if the start_timestamp is not specified, the ingest processes will assume that it's a complete process!